src/components/FilterList/FilterList.d.ts

```diff
@@ -4,6 +4,7 @@ interface FilterType {
   label: string;
   value: string;
   removable?: boolean;
+  [index: string]: any;
```

Comment:
`{ label: 'hello', value: 'world', myVal: 'value' }` should already be compatible without this line due to TS's structural type system, is it not the case for you?

Reply:
Thank you! I had a different issue!